### PR TITLE
fix(progress): modify mobile problem

### PR DESCRIPTION
**Pull request by James-9696**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Summary by CodeRabbit

- **New Features**
	- Enhanced state management for the progress component, centralizing property access.
	- Added a new width property to the progress circle and dashboard components, improving rendering control.

- **Bug Fixes**
	- Adjusted property references to improve functionality and performance of the progress bar.

- **Refactor**
	- Streamlined component structure by removing direct prop imports and updating the setup method.

### CODE REVIEW

File: packages/mobile/components/progress/src/mobile.vue [hunk 1]
```diff
@@ -26,7 +26,7 @@
     aria-valuemax="100"
   >
     <div class="tiny-mobile-progress-bar" v-if="type === 'line'">
-      <div class="tiny-mobile-progress-bar__outer" :style="{ height: strokeWidth + 'px' }">
+      <div class="tiny-mobile-progress-bar__outer" :style="{ height: state.strokeWidth + 'px' }">
```
Comment (gimmyhehe) [inline packages/mobile/components/progress/src/mobile.vue:29]: 29行改为了state.strokeWidth 但是31行没有同步修改

File: packages/mobile/components/progress/src/mobile.vue [hunk 2]
```diff
@@ -26,7 +26,7 @@
...
         <div class="tiny-mobile-progress-bar__inner" :style="state.barStyle">
           <div class="tiny-mobile-progress-bar__innerText" v-if="showText && textInside && strokeWidth > 12">
```
Comment (gimmyhehe) [inline packages/mobile/components/progress/src/mobile.vue]: image
看了一下state.strokeWidth的逻辑，感觉state.strokeWidth没有必要，使用strokeWidth，然后将默认值改为constants.DEFAULT_STROKE_WIDTH就OK了

File: packages/mobile/components/progress/src/mobile.vue (hunk 1 above)
Comment (James-9696) [reply packages/mobile/components/progress/src/mobile.vue:29]: 29行的代码应该使用state.strokeWidth，直接用props的strokeWidth的会导致进度条不显示

File: packages/mobile/components/progress/src/mobile.vue (hunk 2 above)
Comment (James-9696) [reply packages/mobile/components/progress/src/mobile.vue]: 已改成state.strokeWidth

**Review by zzcr — APPROVED** [review]

**Pull request merged**